src/component/AddFood.js

```diff
@@ -0,0 +1,43 @@
+import { useState } from "react"
+import React, { Component } from 'react';
+
+
+
+function AddFood (props) {
+
+    const [name, setName] = useState("");
+    const [calories, setCalories] = useState({});
+    const [image, setImage] = useState("");
+
+    
+    const handleSubmit = (e) => {
+        e.preventDefault();
+
+        const newFood = {
+            name: name,
+            calories: calories,
+            image: image
+        }
+
+        props.callbackCreateFood(newFood)
+
+
```

Comment:
After we submit the form -> we want to clear the input fields. Better user experience.

You can reset the state to empty fields.  After you update newFood, we can reset the form on line23 like this

     setName('');
     setCalories('');
     setImage('');